Source/Core/DolphinWX/Config/AudioConfigPane.cpp

```diff
@@ -241,3 +262,11 @@ void AudioConfigPane::PopulateBackendChoiceBox()
   int num = m_audio_backend_choice->FindString(StrToWxStr(SConfig::GetInstance().sBackend));
   m_audio_backend_choice->SetSelection(num);
 }
+
+void AudioConfigPane::AnyBackendWithLatencyControl()
+{
+  std::vector<std::string> backends = AudioCommon::GetSoundBackends();
+  m_any_backend_with_latency_control =
+      std::any_of(backends.cbegin(), backends.cend(),
+                  [](std::string name) { return AudioCommon::SupportsLatencyControl(name); });
```

Comment:
This can likely be simplified to:

```cpp 
std::any_of(backends.cbegin(), backends.cend(), AudioCommon::SupportsLatencyControl);
```

since it's compatible as a unary predicate.